Documentation/README.md

```diff
@@ -36,7 +36,20 @@
 		* Once your file is approved you will be able to see a preview of the data you are submitting to the database.
 		* Verify one last time that the information is correct. Make changes if required.
 		* Submit the course information to the database.
-2. [Change Application Form]
+2. [Open/Close the TA Application]
+	* Function: This page allows the AC view the application page which is what the student's see. Additionally, this page allows the AC to change the status of the application page between "Closed" and "Open". 
+	* Scenarios for Use: 
+		** 1 - The AC who wants to change the status of the application. 
+		** 2 - The AC wants to view the content of the form, including the front matter if it gets updated from the "Change Application Form" step.
+		** 3 - Students when they want to submit an application. It is important to note that they will NOT have access to the Open/Close funtionality from Scenario 1.
+	* How to Use If You Are the AC: 
+		* Refer to Step 0
+		* Open Step 3, Open/Close the TA Application - (https://team4.uwaterloo.ca/taform/application.html)
+		* Press "Change Status" to change the status of the application. 
+		* When the first line "Application Status" is red and says "Closed", then the form is not live and students can not apply or access the application page.
```

Comment:
Students can access the page, they just get a message saying "the application is closed at this time". Maybe change the wording?